package-lock.json

```diff
@@ -4944,16 +4941,12 @@
     },
     "node_modules/fsevents/node_modules/aproba": {
       "version": "1.2.0",
-      "resolved": "https://registry.npmjs.org/aproba/-/aproba-1.2.0.tgz",
-      "integrity": "sha512-Y9J6ZjXtoYh8RnXVCMOU/ttDmk1aBjunq9vO0ta5x85WDQiQfUF9sIPBITdbiiIVcBo03Hi3jMxigBtsddlXRw==",
       "inBundle": true,
       "license": "ISC",
       "optional": true
     },
     "node_modules/fsevents/node_modules/are-we-there-yet": {
       "version": "1.1.5",
-      "resolved": "https://registry.npmjs.org/are-we-there-yet/-/are-we-there-yet-1.1.5.tgz",
```

Comment:
@fluiddot, I'm not clear why these fields have been removed, could you help me to understand?

Reply:
Good observation. To be honest, I'm not completely sure why they got removed. It might be related to the node/npm version I used when installing the dependencies (v18.14.0 (npm v9.6.1)). I've just tried using node 16 and seems that those fields are back 🤔. I'll edit the commits and push them again. Thanks!

Reply:
Note that I couldn't use node 14 (v14.21.3 (npm v6.14.18)), as the `lockfileVersion` of `package-lock.json` file is `2`. This lock version is compatible with npm 7 and above ([reference](https://docs.npmjs.com/cli/v9/configuring-npm/package-lock-json#lockfileversion)).

https://github.com/wordpress-mobile/react-native-hsv-color-picker/blob/b1c37998b7da0f0d4c7349a6049ffa8597c3557e/package-lock.json#L4

Reply:
> I've just tried using node 16 and seems that those fields are back 🤔. I'll edit the commits and push them again. Thanks!

Seems it's not related to this, as I experienced it again when removing `node_modules` and installing the dependencies. Besides, I also reproduced it using the `wp-fork` branch, so something is weird with the lock files.

In the PR I mentioned that the original repository doesn't use Yarn. Not sure why we introduced it when creating the fork., but I'm thinking of, instead of removing the `yarn.lock` file, removing `package-lock.json` and relying only upon Yarn as it was introduced later in the repo.

Reply:
@SiobhyB I updated the commits and, as mentioned above, now we only rely upon Yarn. Let me know if you could take another look. Thanks!

Reply:
My best guess is that Yarn was introduced to the forked repo during the period of time mobile Gutenberg relied upon Yarn. I believe that was pre-mono-repo-merge. 

I feel `are-we-there-yet` is quite notorious for changing in an obscure, unexpected manner. The approach of `rm -rf node_modules && npm install` generally results in the correct outcome, so it could be that an update to the file is accurate. Maybe the update was overlooked previously.

While odd, I think it is fine to retain Yarn for the time being, particularly if the long-term goal is to remove the fork entirely.

Reply:
> My best guess is that Yarn was introduced to the forked repo during the period of time mobile Gutenberg relied upon Yarn. I believe that was pre-mono-repo-merge.

Yep, that makes sense.

> I feel `are-we-there-yet` is quite notorious for changing in an obscure, unexpected manner. The approach of `rm -rf node_modules && npm install` generally results in the correct outcome, so it could be that an update to the file is accurate. Maybe the update was overlooked previously.

I also thought that might have been overlooked, although not completely sure.

> While odd, I think it is fine to retain Yarn for the time being, particularly if the long-term goal is to remove the fork entirely.

Exactly, eventually, we'll remove the fork so probably it's not critical to keep using Yarn. As a side note, I noticed that most of the forked repositories we use in Gutenberg Mobile use Yarn.